business/MetadataDatabase/Convertor/QidoSeriesConvertor.cs

```diff
@@ -29,7 +29,8 @@ public static SeriesDto ToDto(this QidoSeries series)
                 StudyInstanceUID = series.GetValueOfDicomTag(QidoSeries.DicomTag.StudyInstanceUID),
                 StudyID = series.GetValueOfDicomTag(QidoSeries.DicomTag.StudyID),
                 SeriesNumber = series.GetValueOfDicomTag(QidoSeries.DicomTag.SeriesNumber),
-                NumberOfSeriesRelatedInstances = series.GetValueOfDicomTag(QidoSeries.DicomTag.NumberOfSeriesRelatedInstances)
+                NumberOfSeriesRelatedInstances = series.GetValueOfDicomTag(QidoSeries.DicomTag.NumberOfSeriesRelatedInstances),
+                BodyPartExamined = series.GetValueOfDicomTag(QidoSeries.DicomTag.BodyPartExamined)
```

Comment:
A new field. (To add in the integ tests too. I'll do it after mergind them on develop.)

Reply:
Ok